packages/core/package.json

```diff
@@ -19,7 +19,7 @@
   "license": "MIT",
   "dependencies": {
     "@bugsnag/cuid": "^3.0.0",
-    "@bugsnag/safe-json-stringify": "^2.1.0",
+    "@bugsnag/safe-json-stringify": "3.0.0-0",
```

Comment:
once the PR on @bugsnag/safe-json-stringify is merged and published we should update to a non prerelease version